cloudwash/cli.py

```diff
@@ -98,8 +98,9 @@ def azure(ctx, vms, discs, nics, images, pips, _all, _all_rg):
 @click.option("--images", is_flag=True, help="Remove only images from the provider")
 @click.option("--pips", is_flag=True, help="Remove only Public IPs from the provider")
 @click.option("--stacks", is_flag=True, help="Remove only CloudFormations from the provider")
+@click.option("--ocps", is_flag=True, help="Remove only unused OCPs from the provider")
```

Comment:
```suggestion
@click.option("--ocps", is_flag=True, help="Remove only unused OCP Cluster occupied resources from the provider")
```

How does this sound ?